plugins/doc_fragments/compose_v2.py

```diff
@@ -11,72 +11,67 @@
+      - Mutually exclusive with O(definition).
+    type: list
+    elements: path
+    version_added: 3.7.0
+  definition:
+    description:
+      - Compose file describing one or more services, networks and volumes.
+      - Mutually exclusive with O(project_src) and O(files). One of O(project_src) and O(definition) must be provided.
+      - If provided, PyYAML must be available to this module, and O(project_name) must be specified.
+      - Note that a temporary directory will be created and deleted afterwards when using this option.
+    type: dict
+    version_added: 3.9.0
+  env_files:
+    description:
+      - By default environment files are loaded from a C(.env) file located directly under the O(project_src) directory.
+      - O(env_files) can be used to specify the path of one or multiple custom environment files instead.
+      - The path is relative to the O(project_src) directory.
+    type: list
+    elements: path
+  profiles:
+    description:
+      - List of profiles to enable when starting services.
+      - Equivalent to C(docker compose --profile).
+    type: list
+    elements: str
+  check_files_existing:
+    description:
+      - If set to V(false), the module will not check whether one of the files C(compose.yaml), C(compose.yml), C(docker-compose.yaml),
+        or C(docker-compose.yml) exists in O(project_src) if O(files) is not provided.
+      - This can be useful if environment files with C(COMPOSE_FILE) are used to configure a different filename. The module
+        currently does not check for C(COMPOSE_FILE) in environment files or the current environment.
+    type: bool
+    default: true
+    version_added: 3.9.0
 requirements:
   - "PyYAML if O(definition) is used"
 notes:
-  - |-
-    The Docker compose CLI plugin has no stable output format (see for example U(https://github.com/docker/compose/issues/10872)),
-    and for the main operations also no machine friendly output format. The module tries to accomodate this with various
-    version-dependent behavior adjustments and with testing older and newer versions of the Docker compose CLI plugin.
-
-    Currently the module is tested with multiple plugin versions between 2.18.1 and 2.23.3. The exact list of plugin versions
-    will change over time. New releases of the Docker compose CLI plugin can break this module at any time.
-'''
+  - "The Docker compose CLI plugin has no stable output format (see for example U(https://github.com/docker/compose/issues/10872)),\n
+    and for the main operations also no machine friendly output format. The module tries to accomodate this with various\n
+    version-dependent behavior adjustments and with testing older and newer versions of the Docker compose CLI plugin. Currently
+    the module is tested with multiple plugin versions between 2.18.1 and 2.23.3. The exact list of plugin versions\nwill
+    change over time. New releases of the Docker compose CLI plugin can break this module at any time."
```

Comment:
@russoz looks like andebox makes `|` formatting worse:
```suggestion
  - |-
    The Docker compose CLI plugin has no stable output format (see for example U(https://github.com/docker/compose/issues/10872)),
    and for the main operations also no machine friendly output format. The module tries to accomodate this with various
    version-dependent behavior adjustments and with testing older and newer versions of the Docker compose CLI plugin.
    Currently the module is tested with multiple plugin versions between 2.18.1 and 2.23.3. The exact list of plugin versions
    will change over time. New releases of the Docker compose CLI plugin can break this module at any time.
```